src/Web/Slack.hs

```diff
@@ -213,6 +218,22 @@ chatPostMessage_ ::
   Chat.PostMsgReq ->
   ClientM (ResponseJSON Chat.PostMsgRsp)
 
+-- | Updates a message.
+--
+-- <https://api.slack.com/methods/chat.update>
+chatUpdate ::
+  SlackConfig ->
+  Chat.UpdateReq ->
+  IO (Response Chat.UpdateRsp)
+chatUpdate = flip $ \updateReq -> do
+  authR <- mkSlackAuthenticateReq
+  run (chatUpdate_ authR updateReq) . slackConfigManager
+
+chatUpdate_ ::
```

Comment:
I feel like I've asked this before about this file, but where are all these "underscored" functions defined?

Reply:
https://github.com/MercuryTechnologies/slack-web/blob/ffeccec09c642b71bf6fee4516e15335c8dfc123/src/Web/Slack.hs#L296-L304